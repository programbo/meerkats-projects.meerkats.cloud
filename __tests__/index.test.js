/* global it, expect, describe */

// import React from 'react'
// import { shallow } from 'enzyme'
import renderer from 'react-test-renderer'
import ReactShallowRenderer from 'react-test-renderer/shallow'

import App from '~/pages/index.js'

describe('With Enzyme', () => {
  it('App shows "All projects"', () => {
    const renderer = new ReactShallowRenderer()
    renderer.render(<App />)
    const app = renderer.getRenderOutput()
    expect(app.props.children).toEqual(<h1>All Projects</h1>)
  })
})

describe('With Snapshot Testing', () => {
  it('renders correctly', () => {
    const component = renderer.create(<App />)
    const tree = component.toJSON()
    expect(tree).toMatchSnapshot()
  })
})
