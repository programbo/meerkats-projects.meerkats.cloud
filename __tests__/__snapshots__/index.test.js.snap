// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`With Snapshot Testing renders correctly 1`] = `
<div>
  <div>
    <div
      className="cTmgId"
    >
      <div
        className="hiBmdg"
      >
        <span
          className="fa fa-refresh KdEXZ"
          onMouseEnter={[Function]}
          onMouseLeave={[Function]}
          style={
            Object {
              "MozUserSelect": "none",
              "WebkitTransition": "all 450ms cubic-bezier(0.23, 1, 0.32, 1) 0ms",
              "WebkitUserSelect": "none",
              "color": "#eee",
              "display": "inline-block",
              "fontSize": "20rem",
              "msUserSelect": "none",
              "muiPrepared": true,
              "position": "relative",
              "transition": "all 450ms cubic-bezier(0.23, 1, 0.32, 1) 0ms",
              "userSelect": "none",
            }
          }
        />
      </div>
      <h1
        className="zZUbt"
      >
        Loading...
      </h1>
    </div>
  </div>
</div>
`;
